policyengine_us/parameters/gov/states/ne/tax/income/deductions/standard/base_amount.yaml

```diff
@@ -13,13 +13,16 @@ metadata:
   - title: Nebraska Revised Statute | chapter 77 | §2716.01
     href: https://nebraskalegislature.gov/laws/statutes.php?statute=77-2716.01
   - title: 2023 NE Individual Income Tax Booklet
-    href: https://revenue.nebraska.gov/sites/revenue.nebraska.gov/files/doc/tax-forms/2023/incometax/f_1040n_booklet_2023_Final.pdf#page=9  
+    href: https://revenue.nebraska.gov/sites/revenue.nebraska.gov/files/doc/tax-forms/2023/incometax/f_1040n_booklet_2023_Final.pdf#page=9
+  - title: 2024 Nebraska Individual Income Tax Booklet, Line 6
+    href: https://revenue.nebraska.gov/sites/default/files/doc/tax-forms/2024/f_Individual_Income_Tax_Booklet.pdf#page=10
 
 JOINT:
   values:
     2021-01-01: 14_200
     2022-01-01: 14_700
     2023-01-01: 15_800
+    2024-01-01: 16_700
   uprating:
```

Comment:
could you move the uprating to the general metadata and use `propagate_metadata_to_children` while you are here

Reply:
for all `breakdown` parameters